app/views/comments/edit.html.erb

```diff
@@ -0,0 +1 @@
+<h1> edit a commment in post</h1>
```

Comment:
- [ ] Kindly remove all the **comments** and **likes** views folders. They have no corresponding routes or controller actions, so there is no point in creating them.

  ![image](https://github.com/AliShahSafdari/blog-app/assets/36542986/1f96c473-24dd-4a2d-bc9d-679fe76197f4)